ui/src/app/base/components/FormikField/FormikField.tsx

```diff
@@ -4,12 +4,16 @@ import PropTypes from "prop-types";
 import React, { useRef } from "react";
 import { nanoid } from "@reduxjs/toolkit";
 
+import { TSFixMe } from "app/base/types";
+
+type Props = { [x: string]: TSFixMe };
```

Comment:
I was getting a weird error in .tsx files that imported any Formik component. To get around it I've converted them to tsx and added a props type that is an object with any key-value pairs.

Reply:
Hey @Caleb-Ellis, can you clarify what you mean by "might be using Formik wrong"?

Reply:
@squidsoup Not really hahaha, it was just an off-handed comment where basically I think we should revisit how we've built our custom Formik components (FormikForm, FormikFormButtons, FormikFormContent and FormikField). There's a few hooks provided by Formik that we could use to reduce the amount of props getting passed 3+ components down.